docs/source/markdown/podman-run.1.md

```diff
@@ -751,6 +751,12 @@ By default, Podman will manage _/etc/hosts_, adding the container's own IP addre
 #### **--no-hosts** disables this, and the image's _/etc/hosts_ will be preserved unmodified.
 This option conflicts with **--add-host**.
 
+#### **--no-manage-passwd**
+
+Prohibits podman from creating entries in /etc/passwd or /etc/groups when used in conjunction with --user.
```

Comment:
nit 's/creating/adding/g'

Reply:
so you mean `Prohibit podman from adding entries in /etc/passwd or /etc/group when used in conjunction with --user.`

Reply:
I want 
#### **--passwd**

Allow podman to add entries to /etc/passwd and /etc/groups when used in conjunction with --user option.